### DISCUSSION TITLE
Arrows anchored at their center

### DISCUSSION THREAD

**Discussion by avaruuser** [pyvista/pyvista#2596] [Q&A] [answered]
Hello, 

I'm wondering if it is possible to anchor the arrow glyphs to their center such that they rotate around their midpoint instead of the end of the arrow-shaft. I would appreciate any help with this.

**Comment by MatthewFlamm** [comment, marked as answer]
This example centers the arrow from tip to tip, the blue sphere is centered at the same point that the glyph was generated at. It looks non centered to my eye because of the cone shape on one side.  This could be fiddled with if an optically pleasing center is preferred


```py
import pyvista
sphere = pyvista.Sphere(radius=0.1)
arrow = pyvista.Arrow(start=(-0.5, 0, 0))
point = pyvista.PolyData([0., 0., 0.])
point["orient"] = [[0., 1., 0.]]
glyph = point.glyph(geom=arrow)
plotter = pyvista.Plotter()
plotter.add_mesh(sphere, 'blue')
plotter.add_mesh(glyph, 'red')
plotter.show_axes()
plotter.show()
```


![image](https://user-images.githubusercontent.com/39341281/168333195-23a888d1-c154-4b19-87cf-cfd776fd6633.png)

**Reply by avaruuser** [reply]
Thank you so much, that is exactly what I needed.
